### PR TITLE
Add Vela core helm chart

### PR DESCRIPTION
**Pull request by ryanzhang-oss**

### CODE REVIEW

**Comment by ryanzhang-oss** [comment]
The podSpec doesn't work with 1.18, the CRD generator cannot generate 1.18 compatible CRD

https://github.com/kubernetes-sigs/controller-tools/issues/444

**Comment by ryanzhang-oss** [comment]
- Still need to install Prometheus/Grafana
- Need to make webhook work (cert-manager hook complained)

File: charts/third_party/istio-stable/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+The istio\*.yaml files are generated by running
+
+```
+./download-istio.sh
```
Comment (wonderflow) [inline charts/third_party/istio-stable/README.md:4]: why do we need to install istio?

Comment (ryanzhang-oss) [reply charts/third_party/istio-stable/README.md:4]: will remove

File: charts/vela/crds/standard.oam.dev_containerizeds.yaml [hunk 2]
```diff
@@ -0,0 +1,5768 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
```
Comment (wonderflow) [inline charts/vela/crds/standard.oam.dev_containerizeds.yaml:3]: will this CRD generate automatically if we change the code?

**Review by wonderflow — COMMENTED** [review]
Really great work!

File: charts/third_party/cert-manager/owner-ref.patch [hunk 3]
```diff
@@ -0,0 +1,13 @@
+6206@6207,1
+diff --git a/third_party/cert-manager-0.12.0/cert-manager.yaml b/third_party/cert-manager-0.12.0/cert-manager.yaml
+index 1ee179c02..0a96ed86b 100644
+--- a/third_party/cert-manager-0.12.0/cert-manager.yaml
```
[the remark this answers is not in the document]
Comment (ryanzhang-oss) [reply charts/third_party/cert-manager/owner-ref.patch:4]: I don't really know what it does. I will remove it and see.

File: charts/vela/crds/standard.oam.dev_containerizeds.yaml (hunk 2 above)
Comment (ryanzhang-oss) [reply charts/vela/crds/standard.oam.dev_containerizeds.yaml:3]: yes, the CRDs whose type are defined in our repo will change. The others are copied.

Comment (wonderflow) [reply charts/vela/crds/standard.oam.dev_containerizeds.yaml:3]: cool

**Review by wonderflow — APPROVED** [review]

**Pull request merged**